ios/RNSelectableTextView.m

```diff
@@ -116,4 +163,20 @@ - (void)forwardInvocation:(NSInvocation *)invocation
     }
 }
 
+- (BOOL)canBecomeFirstResponder
+{
+    return YES;
+}
+
+- (BOOL)canPerformAction:(SEL)action withSender:(id)sender
+{
+    NSString *sel = NSStringFromSelector(action);
+    NSRange match = [sel rangeOfString:@"_CUSTOM_SELECTOR_"];
```

Comment:
could we have _CUSTOM_SELECTOR as a constant and use the constant instead? just for the sake of safety